ambry-router/src/main/java/com.github.ambry.router/GetBlobOperation.java

```diff
@@ -84,10 +88,15 @@
   private int numChunksTotal;
   // the total number of data chunks retrieved so far (and may or may not have been written out yet).
   private int numChunksRetrieved;
+  // the offset of the first chunk index in the requested byte range
```

Comment:
why not just say "the index/position of the first chunk in the requested byte range" - would prefer that offsets are only used to refer to "byte offsets" at least in the context of these classes.